package/MDAnalysis/analysis/dssp/pydssp_numpy.py

```diff
@@ -0,0 +1,267 @@
+"""Default numpy implementation for pydssp that doesn't use `einops` package"""
```

Comment:
add more information on the original source (include author and copyright as found in original repo, include URL) and summarize the changes to the file.

Reply:
Done.